pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_unpublishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,22 @@
+include::../attributes.adoc[]
+[id="unpublishing-a-module-in-pantheon_{context}"]
+= Unpublishing a module in {ProductShortName}
+
+Unpublish a module in {ProductShortName} from the Customer Portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to unpublish.
++
+If you don't see the module listed, search for the modules using the search box.
+
+. Click *Unpublish* next to the module version you want to unpublish.
```

Comment:
module version => version of the module

Sounds better to me. Suggestion.

Reply:
Agree, updated.

Reply:
I don't see this in 0a83c34.

Reply:
The change is there at https://github.com/redhataccess/pantheon/pull/254/files#diff-63429acfd56fbee86c05f1530069352fR20

Reply:
LGTM